hubmap/frontend/cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blacklistHosts": ["localhost:8000"],
```

Comment:
This may work, but I think we'd be better off not depending on Django in Cypress. This is subjective, but I think it will be better to use cypress just for testing only the client side code, and all API responses we would get are done with fixtures instead.

Reply:
Refer: https://github.com/hubmapconsortium/hubmap-data-portal/pull/103#discussion_r315256409

Reply:
@mccalluc  However, this is removed. But this is this is one of reasons, I requested we need to document this for testing and requested your suggestion on my reply last week. https://github.com/hubmapconsortium/hubmap-data-portal/issues/98